src/com/mambu/apisdk/MambuAPIFactory.java

```diff
@@ -58,8 +59,25 @@
 	 * @param password
 	 *            the password used by the user
 	 */
+	@Deprecated
```

Comment:
We can remove @Deprected and just change the description to "Convenience" method.  Current API users then would  not be needed to make any changes with this enhancement.